ListOfQuestions.md

```diff
@@ -7,4 +7,11 @@
           [:a :b :c]
           [:a :b :c]  => (list :a :b :c) (vec '(:a :b :c)) (vector :a :b :c))])
 
-* wrap block with parentsesis
\ No newline at end of file
+* wrap block with parentsesis
+
+(defn	alice-is	[input]
+	->>	(loop	[in	input  out	[]] <--
```

Comment:
it's just a question how this pattern match works. However, I've figure out it already ;)